plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTablePqlExtractor.java

```diff
@@ -113,4 +116,9 @@ public static String encloseInParentheses(String value)
     {
         return format("(%s)", value);
     }
+
+    private static String quoteIdentifier(String identifier)
+    {
+        return format("\"%s\"", identifier);
```

Comment:
`checkArgument` that identifier doesn't contain `"` (or escape it properly for Pinot)